components/Footer.js

```diff
@@ -47,29 +47,29 @@ export default function Footer() {
         </nav>
         <div className="mt-8 flex justify-center">
           <a href="https://twitter.com/AsyncAPISpec" target="_blank" rel="noopener noreferrer" className="text-gray-400 hover:text-blue-500">
-            <span className="sr-only">Twitter</span>
+            <span className="sr-only">Follow AsyncAPI on Twitter</span>
```

Comment:
I'm particularly not sure of this because it doesn't add to any title or alt text for image here. Same goes with other icons as well. And may be this change won't increase the accessibility and performance of the website.